one/src/events.rs

```diff
@@ -113,42 +115,45 @@ async fn validate(opts: ValidateOpts) -> Result<()> {
     //
     // When validating events from Recon, the block store will be a CARFileBlockStore that wraps the CAR file received
     // over Recon.
-    if let Some(block) = block_store.get(opts.cid).await? {
-        let event = CborValue::parse(&block)?;
-        if event.get_key("proof").is_some() {
-            let timestamp = validate_time_event(
-                &opts.cid,
-                &block_store,
-                &root_store,
-                &HttpEthRpc::new(opts.ethereum_rpc_url),
-            )
-            .await;
-            match timestamp {
-                Ok(timestamp) => {
-                    let rfc3339 = Utc
-                        .timestamp_opt(timestamp, 0)
-                        .unwrap()
-                        .to_rfc3339_opts(SecondsFormat::Secs, true);
-                    println!(
-                        "TimeEvent({}) validated at Timestamp({}) = {}",
-                        opts.cid, timestamp, rfc3339
-                    )
-                }
-                Err(e) => println!("{} failed with error: {:?}", opts.cid, e),
-            }
-        } else {
-            if event.get_key("data").is_some() {
-                println!("UnsignedDataEvent({}) is not signed", opts.cid);
+    let block = block_store.get(&opts.cid).await?;
+    let event = CborValue::parse(&block.data)?;
+    if event.get_key("proof").is_some() {
+        let timestamp = validate_time_event(
+            &opts.cid,
+            &block_store,
+            &root_store,
+            &HttpEthRpc::new(opts.ethereum_rpc_url),
+        )
+        .await;
+        match timestamp {
+            Ok(timestamp) => {
+                let rfc3339 = Utc
+                    .timestamp_opt(timestamp, 0)
+                    .unwrap()
+                    .to_rfc3339_opts(SecondsFormat::Secs, true);
+                println!(
```

Comment:
log?

Reply:
we didn't have tracing in the cli yet.. I setup a subscriber now and changed almost all of the stdout/stderr messaging to use info/warn/error.